NetKAN/PartsUnlimited.netkan

```diff
@@ -0,0 +1,21 @@
+{
+  "license": "CC-BY",
+  "identifier": "PartsUnlimited",
+  "x_via": "Automated Linuxgurugamer CKAN script",
+  "$kref": "#/ckan/spacedock/927",
+  "depends": [
+    {
+      "name": "ModuleManager"
+    },
+    {
+      "name": "ContractConfigurator"
```

Comment:
From the notes on Spacedock, this needs to have `, "min_version": "1.18.2"` added